californium-core/src/main/java/org/eclipse/californium/core/network/stack/ReliabilityLayerParameters.java

```diff
@@ -0,0 +1,110 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Lari Hotari and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ *
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ *
+ * Contributors:
+ *     Lari Hotari - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.californium.core.network.stack;
+
+import org.eclipse.californium.core.network.config.NetworkConfig;
+
+/**
+ * Contains the configuration values for {@link ReliabilityLayer} and {@link CongestionControlLayer}
+ */
+public class ReliabilityLayerParameters {
```

Comment:
should we call this class TransmissionParameters ? and add a reference to this url in the comment https://tools.ietf.org/html/rfc7252#section-4.8 ?